producer/tracing/spiders/main.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+import time
+
+import boto3
+import requests
+import scrapy
+from ddtrace import tracer
+
+
+@tracer.wrap("producer.send")
+def send_to_queue(content):
```

Comment:
I suggest moving the logic of this function into TutorialPipeline::process_item, it's more similar to how scrapy would actually be used, with the spider returning items and the scrapy pipeline taking care of the forwarding.